.github/CODEOWNERS

```diff
@@ -102,6 +102,7 @@ core/scripts/gateway @smartcontractkit/functions
 
 # CI/CD
 /.github/** @smartcontractkit/releng @smartcontractkit/test-tooling-team
+/.github/CODEOWNERS @smartcontractkit/prodsec-public @smartcontractkit/foundations
```

Comment:
@chainchad should we (@smartcontractkit/releng) be added here as well?

Reply:
If we wanted to, we could switch the order of these two lines but I don't think it's necessary as most of the changes to CODEOWNERS would be managed by Foundations who would be in the best position to understand who should own new paths.